### PR TITLE
1.1 Update grunt doc

### PR DESCRIPTION
**Pull request by ngilain**

### CODE REVIEW

File: en/developer_guide/manage_assets.rst [hunk 1]
```diff
@@ -4,187 +4,179 @@ Manage assets with Grunt
 Context
 -------
 
-To automate all the assets management process, the Open Orchestra Back Office uses the task manager
-`Grunt`_.
+To automate all the assets management process, the Open Orchestra Back Office uses the task manager `Grunt`_.
+This tool allows the creation and execution of assets related tasks in the nodeJs format. For instance Open
```
Comment (alavieille) [inline en/developer_guide/manage_assets.rst]: nodeJs format ?

File: en/developer_guide/manage_assets.rst [hunk 2]
```diff
@@ -4,187 +4,179 @@ Manage assets with Grunt
...
+Orchestra uses it to compile coffee scripts or to create some symlinks.
```
Comment (alavieille) [inline en/developer_guide/manage_assets.rst]: CoffeeScript

File: en/developer_guide/manage_assets.rst [hunk 3]
```diff
@@ -4,187 +4,179 @@ Manage assets with Grunt
...
 
-*Grunt* is looking for its configuration inside the ``Gruntfile.js`` file in the root directory of
-the project. This file mainly parses configuration files to recreate a Grunt configuration object.
+When running the command
```
Comment (alavieille) [inline en/developer_guide/manage_assets.rst]: why not just `when running grunt` ?

File: en/developer_guide/manage_assets.rst [hunk 4]
```diff
@@ -4,187 +4,179 @@ Manage assets with Grunt
...
-When you install your Open Orchestra Back Office, you have to create a ``Gruntfile.js`` to manage
-the assets included in the bundles you choose. This file loads and parses configuration files to
-produce the Grunt configuration.
+When you install the Open Orchestra Back Office, you have to create a ``Gruntfile.js`` to manage the assets
+included in the bundles you are using. This file loads and parses configuration files to produce the Grunt
+configuration.
 
-You are totally free to code this the way you prefer, and here is a functionnal sample providing
-standard functionnalities:
+As this file belongs to your application, you are totally free to code it the way you want. But to avoid waste
```
Comment (alavieille) [inline en/developer_guide/manage_assets.rst]: may be specify more specifically this part

File: en/developer_guide/manage_assets.rst [hunk 5]
```diff
@@ -4,187 +4,179 @@ Manage assets with Grunt
...
+here where you register your bundles providing new *Grunt* tasks.
 
-This bundle comes with its own javascript and css files that need to be added to the website assets
-files. Assuming you have installed the bundle using composer, you still have to configure Grunt to
-use the bundle assets.
+The targetsDir attribute is an array listing all folders including *Grunt* task targets. This is typically
+here where you register your bundles providing new *Grunt* tasks.
 
-For that purpose, you need to update 3 parts of the Grunt configuration:
+For this little mechanism to work properly, tasks and target have to be normalized with a little few rules:
```
Comment (alavieille) [inline en/developer_guide/manage_assets.rst]: may be remove little

**Comment by ngilain** [comment]
@itkg-alavieille : update done
@itkg-nanne , @Thiblef : a PR on doc still requires 2 validation :)

File: en/developer_guide/manage_assets.rst [hunk 6]
```diff
@@ -4,187 +4,179 @@ Manage assets with Grunt
...
+Orchestra uses it to compile *CoffeeScript* files or to create some symlinks.
 
-*Grunt* is looking for its configuration inside the ``Gruntfile.js`` file in the root directory of
-the project. This file mainly parses configuration files to recreate a Grunt configuration object.
+When running grunt via the command
 
-In order to simplify the management of external bundles assets, each bundle describes its own
-Grunt task and/or task options. As a result, adding a new bundle will simply require little change
-in the ``Gruntfile.js`` of your project to load the new configuration parts.
+.. code-block:: javascript
```
Comment (nbouquet) [inline en/developer_guide/manage_assets.rst]: code-block bash plutôt

File: en/developer_guide/manage_assets.rst [hunk 7]
```diff
@@ -4,187 +4,179 @@ Manage assets with Grunt
...
+The targetsDir attribute is an array listing all folders including *Grunt* task targets. This is typically
+here where you register your bundles providing new *Grunt* tasks.
 
-For that purpose, you need to update 3 parts of the Grunt configuration:
+For this mechanism to work properly, tasks and target have to be normalized with a few rules:
 
-- load the new tasks
-- load the new task options
-- tell somewhere when these tasks must be run
+* First of all, they must be written as AMD modules that the GruntConfigBuilder will load
```
Comment (veolys-nanne) [inline en/developer_guide/manage_assets.rst]: maybe could you add a link to AMD norm?

**Comment by veolys-nanne** [comment]
:+1:

File: en/developer_guide/manage_assets.rst [hunk 8]
```diff
@@ -4,187 +4,179 @@ Manage assets with Grunt
...
-produce the Grunt configuration.
+When you install the Open Orchestra Back Office, you have to create a ``Gruntfile.js`` to manage the assets
+included in the bundles you are using. This file loads and parses configuration files to produce the Grunt
+configuration.
 
-You are totally free to code this the way you prefer, and here is a functionnal sample providing
-standard functionnalities:
+The ``Grunfile.js`` file belongs to your application, so it's up to you to write it. But to avoid you a waste
+of time, you can use the GruntConfigBuilder AMD module developped for Open Orchestra to load *Grunt* splitted
+configuration. The file is packaged with the [open-orchestra-cms-bundle](https://github.com/open-orchestra/open-orchestra-cms-bundle)
```
Comment (alavieille) [inline en/developer_guide/manage_assets.rst]: uses rst syntax to create link

File: en/developer_guide/manage_assets.rst [hunk 9]
```diff
@@ -4,187 +4,179 @@ Manage assets with Grunt
...
+and located in the GruntTasks folder. The [Open Orchestra Back Office demo project](https://github.com/open-orchestra/open-orchestra)
```
Comment (alavieille) [inline en/developer_guide/manage_assets.rst]: same for the link

**Comment by ngilain** [comment]
@itkg-nanne , @itkg-alavieille : update done

**Comment by alavieille** [comment]
:+1:

**Pull request merged**